presidio-analyzer/Pipfile

```diff
@@ -4,17 +4,17 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-spacy = "==3.0.6"
+spacy = "==3.2.0"
```

Comment:
Can we allow more flexibility here?

Reply:
I thought that for the dependencies changing the detections regularly, at least in the previous bumps we did, to lock it to minor.

Reply:
Anyway, done